docs/en/data_types/domains/domains.md

```diff
@@ -0,0 +1,24 @@
+# Domains
+
+Domains are special-purpose types, that add some extra features atop of existing 'base' type, leaving on-wire and on-disc format of underlying table intact. At the moment, Clickhouse does not support user-defined domains, and builtin domains are limited in functionality.
+
+You can use domains in anywhere corresponding base type can be used:
+
+* Create a column of domain type
+* Read/write values from/to domain column
+* Use it as index if base type can be used as index
+* Call functions with values of domain column
+* etc.
+
+### Extra features of domains
+
+* Explicit column type name in `SHOW CREATE TABLE` or `DESCRIBE TABLE`
+* Custom input format for `INSERT INTO domain_table(domain_column) VALUES(...)`
+* Custom output format for `SELECT domain_column FROM domain_table`
+* Loading data from external source: `INSERT INTO domain_table FORMAT CSV ...`
```

Comment:
Actually these are called input/output formats in the rest of documentation, not the features from previous two bullets. Those are more like "Input from human-readable representation for INSERT INTO..." and "Output to human-readable representation for SELECT...".

Reply:
Fixed